CONTRIBUTING.md

```diff
@@ -145,10 +128,10 @@ scripting APIs, you **must** update the class reference to document those.
 This is to ensure the documentation coverage doesn't decrease as contributions
 are merged.
 
-[Update documentation XML files](https://docs.godotengine.org/en/latest/contributing/documentation/updating_the_class_reference.html)
+[Update documentation XML files](https://docs.blaziumengine.org/en/latest/contributing/documentation/updating_the_class_reference.html)
```

Comment:
I think we have https://blazium.app/ now... not sure If bio plans to host a "docs" subdomain on it yet though!